### PR TITLE
chore: update HTML lectures intros

### PR DESCRIPTION
**Pull request by Ksound22**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by majestic-owl448 — APPROVED** [review]

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1890,7 +1890,9 @@
       },
       "lecture-working-with-tables": {
         "title": "Working with Tables",
-        "intro": ["In these lecture videos, you will learn about HTML tables."]
+        "intro": [
+          "In these lecture videos, you will learn about HTML tables, how to create them, what they should be used for, and what they should not be used for."
```
Comment (Dario-DC) [inline client/i18n/locales/english/intro.json:1894]:
```suggestion
          "In these lecture videos, you will learn about HTML tables, how to create them, and when to use them."
```

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -1927,7 +1929,7 @@
       "lecture-importance-of-accessibility-and-good-html-structure": {
         "title": "Importance of Accessibility and Good HTML Structure",
         "intro": [
-          "In these lecture videos, you will learn about importance of accessibility and using good HTML structure."
+          "In these lecture videos, you will learn about accessibility and its importance, assistive tools for people with disabilities, HTML attributes that let you create inclusive websites, accessibility best practices, and many more."
```
Comment (Dario-DC) [inline client/i18n/locales/english/intro.json:1932]:
```suggestion
          "In these lecture videos, you will learn about accessibility and its importance, assistive tools for people with disabilities, HTML attributes that let you create inclusive websites, accessibility best practices, and much more."
```

**Review by zairahira — COMMENTED** [review]
@Ksound22, these changes are looking great.

I was auditing the curriculum and I have one suggestion.

By the time campers reach "The cat Photo app", they would know what HTML stands for. 

Here, the sentence `"HTML stands for HyperText Markup Language and it represents the content and structure of a web page."` is looking a bit repetitive.

image

**Comment by Ksound22** [comment]
> @Ksound22, these changes are looking great.
> 
> I was auditing the curriculum and I have one suggestion.
> 
> By the time campers reach "The cat Photo app", they would know what HTML stands for.
> 
> Here, the sentence `"HTML stands for HyperText Markup Language and it represents the content and structure of a web page."` is looking a bit repetitive.
> 
> image

I'm taking care of lecture intros for now. We'll address that later.

**Comment by Dario-DC** [comment]
> I'm taking care of lecture intros for now. We'll address that later.

Wouldn't it be better to make a PR for each module? Updating lectures, labs and workshops?

**Comment by Ksound22** [comment]
> > I'm taking care of lecture intros for now. We'll address that later.
> 
> Wouldn't it be better to make a PR for each module? Updating intros for lectures, labs and workshops?

Many of the labs and workshops have their descriptions well-written, but I'll look around for whatever I can make better now.

Thank you!

**Comment by Ksound22** [comment]
@Dario-DC 👋

Zaira has opted to take care of the descriptions for the labs.

I think we have to continue with the same pattern. 

But my next PR will take care of the remaining lectures, then the workshops, the reviews, and finally the quizzes.

**Comment by Dario-DC** [comment]
Sure @Ksound22 , mine was just a suggestion :+1:

**Review by Dario-DC — APPROVED** [review]

**Pull request merged**